### PR TITLE
Add options to disable all the things

### PR DESCRIPTION
**Pull request by pixeltrix**
* Disable 'Local to you'
* Disable MSPs on signed page
* Disable 'Register to vote'
* Disable 'Other parliamentary business'
* Disable thresholds and debates

Not everything was adding feature flags:

* Restore 'What do you want us to do?' on the check existing petitions page
* Add option to reset the Gaelic content
* Add 'Collecting signatures' section to the home page
* Ensure creators have validated their email address before petition is published

### CODE REVIEW

File: app/views/admin/petitions/_petition_content.html.erb [hunk 1]
```diff
@@ -38,7 +38,11 @@
   <%= link_to 'Gaelic', admin_petition_details_path(@petition, locale: "gd-GB") %>
 
   <% if Site.disable_gaelic_website? %>
-    <%= button_to 'Copy content to Gaelic petition', copy_content_admin_petition_path(@petition), class: 'button' %>
+    <% if @petition.gaelic_translated? %>
+      <%= button_to 'Remove Gaelic petition content', admin_petition_content_path(@petition), method: :delete, class: 'button button-warning', data: { confirm: 'Are you sure? This will remove any content in the Gaelic fields.'} %>
+    <% else %>
+      <%= button_to 'Copy content to Gaelic petition', admin_petition_content_path(@petition), method: :post, class: 'button', data: { confirm: 'Are you sure? This will copy the content in the English fields over any content in the Gaelic fields.'} %>
```
Comment (freesteph) [inline app/views/admin/petitions/_petition_content.html.erb:44]: do we want the alert copy to live in the translation files?

File: app/views/admin/sites/edit.html.erb [hunk 2]
```diff
@@ -4,22 +4,20 @@
...
-      <%= render "form", tab: "access" %>
-    <% elsif params[:tab] == "features" %>
-      <%= render "form", tab: "features" %>
-    <% else %>
-      <%= render "form", tab: "english" %>
-    <% end %>
-  </div>
 </div>
+
+<% if params[:tab] == "gaelic" %>
```
Comment (freesteph) [inline app/views/admin/sites/edit.html.erb:9]: maybe move this to a controller `current_tab` helper method of some kind?

```rb
def current_tab
  @current_tab = params[:tab] || 'english'
end
```

?

**Review by freesteph — APPROVED** [review]

File: app/views/admin/petitions/_petition_content.html.erb (hunk 1 above)
Comment (pixeltrix) [reply app/views/admin/petitions/_petition_content.html.erb:44]: No, none of the admin stuff other than some lists, etc. are in the locale files.

File: app/views/admin/sites/edit.html.erb (hunk 2 above)
Comment (pixeltrix) [reply app/views/admin/sites/edit.html.erb:9]: Will do later this week

**Pull request merged**